### ISSUE TITLE
[Bug]: Sidebar background in mobile view

### ISSUE THREAD

**Issue by rohinish404** [FrancescoXX/free-Web3-resources#411]
### Description

The background of the navbar in mobile view is not matching with the background of the home page. The sidebar should be white and black/gray background in the respective modes.
<img width="498" alt="Screenshot 2023-03-06 at 10 14 53 AM" src="https://user-images.githubusercontent.com/92542124/223021432-7d7cb099-1653-4ced-9eae-56c3cf7a77dc.png">

### To Reproduce

1. Go to home age in mobile view.
2. Click on menu button on top right.
3. Sidebar appears

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/FrancescoXX/4c-site/blob/main/docs/CODE_OF_CONDUCT.md)

**Comment by tobySolutions** [comment]
Do you want to work on this Roshinish?

**Comment by rohinish404** [comment]
Yeah sure! I'll make the changes. Should i go with the white background and dark gray text.?

**Comment by tobySolutions** [comment]
Yeah, go with that @rohinish404. Thanks!

**Comment by rohinish404** [comment]
Made a pr regarding this :)

**Comment by tobySolutions** [comment]
Great job @rohinish404. Fixed by issue #412

**Comment by rohinish404** [comment]
Thank you @tobySolutions 🙂
